app/src/controllers/review/services/ReviewService.ts

```diff
@@ -1,4 +1,4 @@
-import { OrderBy } from '@entities/Common'
+import { OrderBy, ReviewScore } from '@entities/Common'
```

Comment:
there's already a Review score inside the Review model, i don't see why we have to crate one again in the common?